src/tile-board/Tileboard.jsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import Tile from './Tile';
+
+function TileBoard() {
+  const arrayGrid = createArrayGrid();
```

Comment:
This is going to create a new array each time `<TileBoard />` is rendered. This won't currently be an issue because I don't think your app currently has anything that would cause it to re-render (either the parent `<App />` component re-rendering, or a state change inside of `TileBoard`). However, that will likely change once you add more state and interactivity.

If you only need to create the array grid once, you could just pull this line up outside of the component and into the module scope where it would only be executed once or you could use `useState` to keep a stable reference to the return value of `createArrayGrid()`:

```js
const [grid] = useState(() => createArrayGrid())
```

This is using the lazy initialization pattern for `useState` which you can read more about [here](https://kentcdodds.com/blog/use-state-lazy-initialization-and-function-updates).